app-modules/bot-discord/src/Events/DynamicVoiceEvent.php

```
@@ -0,0 +1,74 @@
+{
+    /**
+     * The event handler.
+     *
+     * @var string
+     */
+    protected $handler = Events::VOICE_STATE_UPDATE;
+
+    /**
+     * Handle the event.
+     */
+    public function handle(VoiceStateUpdate $state, Discord $discord, ?VoiceStateUpdate $oldState): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$discord'. (undefined)

(UnusedFormalParameter)